src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,68 @@
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-minutes} / 2));
```

Comment:
The calculation `calc(50% - (#{$hand-width-minutes} / 2))` is used to center the minutes hand. While this is correct, ensure that the `$hand-width-minutes` variable is defined and has a unit that makes this calculation valid in CSS.